apps/frontend/src/pages/projects/styles.module.css

```
@@ -1,5 +1,8 @@
 .label {
```

class names should also be consistent
```suggestion
.title {
```